src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -502,6 +502,21 @@ protected DynamoModel(IStartConfiguration config)
                 }
             }
 
+            // At this point, pathManager.PackageDirectories only has 1 element which is the directory
+            // in AppData. If list of PackageFolders is empty, add the folder in AppData to the list since there
+            // is no additional location specified. Otherwise, update pathManager.PackageDirectories to include
+            // PackageFolders
+            if (PreferenceSettings.PackageFolders.Count == 0)
+                PreferenceSettings.PackageFolders = new List<string>(pathManager.PackagesDirectories);
+            else
+                pathManager.LoadPackageFolders(PreferenceSettings.PackageFolders);
```

Comment:
The thing is, `PreferenceSettings.PackageFolders` may or may not contain `AppData`, so that won't work.